hieradata/common.yaml

```diff
@@ -248,6 +248,38 @@ profile::pluginsite::image_tag: '24-0d1adf'
 azure::releases::account_name: 'overridewithstorageaccountname'
```

Comment:
We should specify which storage account this variable is used for.
As have different storage account for different reasons.

- tfstate
- k8s
- logs
- ...